.github/workflows/requirements.yaml

```diff
@@ -0,0 +1,22 @@
+name: requirements
+
+on: [pull_request, push]
+
+jobs:
+  requirements:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
```

Comment:
Same question as above, but for different python versions.  I don't think we claim that the notebooks support anything other than 3.7

Reply:
I'm fine with sticking with just 3.7